src/main/java/Main.java

```diff
@@ -1,8 +1,49 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Main {
-
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        startRace();
+    }
+
+    public static void startRace() {
+        System.out.println("Начнем гонку!");
+
+        Wheel newWheel = new Wheel();
+        Scanner scanner = new Scanner(System.in);
+
+        String riderName = "";
+        Integer riderSpeed = 0;
```

Comment:
```suggestion
        int riderSpeed = 0;
```
здесь можно использовать примитив, а не создавать объект класса и выделять доп. память под него